config/config.go

```diff
@@ -554,10 +554,16 @@ var defaultConf = Config{
 		StoreLimit:     0,
 
 		CoprCache: CoprocessorCache{
-			Enabled:               true,
-			CapacityMB:            1000,
-			AdmissionMaxResultMB:  10,
-			AdmissionMinProcessMs: 5,
+			//Enabled:               false,
+			//CapacityMB:            1000,
+			//AdmissionMaxResultMB:  10,
+			//AdmissionMinProcessMs: 5,
```

Comment:
What are these comments for?

Reply:
Some recommended configurations if user want it back.

Reply:
Did  you mean `Enabled:  true`?